intf_.mod

```diff
@@ -677,11 +681,11 @@ PROCEDURE trvsp ()
 {
   VERBATIM 
   int i, flag; 
-  double ind, t0;
+  double ind, t0_local;
```

Comment:
See the above resolved conversation explaining why `t0` uneeds to be renamed to `t0_local`.